Tests/SmithyXMLTests/NamespaceReaderTests.swift

```diff
@@ -0,0 +1,128 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import SmithyXML
+import SmithyReadWrite
+import ClientRuntime
+import XCTest
+
+class NamespaceReaderTests: XCTestCase {
```

Comment:
XML Namespaces required extra debugging.  This is a test that I coped in from protocol tests to aid in debugging namespaces.